TransformationSystem/Client/TaskManager.py

```diff
@@ -584,6 +586,8 @@ def __prepareTasksBulk(self, transBody, taskDict, owner, ownerGroup, ownerDN):
       # Handle Input Data
       inputData = paramsDict.get('InputData')
       if inputData:
+        if isinstance(inputData, basestring):
```

Comment:
@andresailer the only real change related to the release notes is here, in case you missed it.